docs/src/Tabs.doc.js

```diff
@@ -31,7 +31,7 @@ card(
       },
       {
         name: 'onChange',
-        type: `({ event: SyntheticMouseEvent<>, activeTabIndex: number }) => void`,
+        type: `({ +event: SyntheticMouseEvent<> | SyntheticKeyboardEvent<>, +activeTabIndex: number }) => void`,
```

Comment:
@jackhsu978 love the read only change, we should make that the default for other components as well. (out of scope for this PR)

Reply:
yeah, good idea!